platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -797,6 +800,49 @@ public void removeOnStyleImageMissingListener(@NonNull OnStyleImageMissingListen
     mapChangeReceiver.removeOnStyleImageMissingListener(listener);
   }
 
+  /**
+   * Set a callback that's invoked when map needs to release unused image resources.
```

Comment:
It'd be great to mention what's the default behavior and that removing a custom listener after it's been added falls back to the default behavior.

Reply:
Sure, let's wait for @tobrun, I might need to remove commit `[android] Add default handler for onRemoveUnusedStyleImages API` completely, if everyone agrees.

Reply:
Added few notes to the API description.